### PR TITLE
Base style

### PR DESCRIPTION
**Pull request by noobjey**
Using sass with mdl for styling

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<footer class="mdl-mini-footer">
+  <div class="mdl-mini-footer__left-section">
+    <div class="mdl-logo">Dinners Ready 4 U</div>
+    <ul class="mdl-mini-footer__link-list">
+      <li><a href="#">Help</a></li>
```
Comment (applegrain) [inline app/views/layouts/_footer.html.erb]: let's make sure we remember to change this to the actual links/info we want in the footer

File: app/views/layouts/_messages.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<%# Rails flash messages styled for Bootstrap 3.0 %>
```
Comment (applegrain) [inline app/views/layouts/_messages.html.erb]: you know what to do

File: app/assets/stylesheets/application.sass [hunk 3]
```diff
@@ -0,0 +1,19 @@
+/*
```
Comment (applegrain) [inline app/assets/stylesheets/application.sass]: feel free to delete the comments

**Comment by applegrain** [comment]
closes #4 
Looks good! Make the changes and merge.

**Comment by mrjaimisra** [comment]
closes #4 
nice work JaySON!

**Pull request merged**